app/api/rating.py

```diff
@@ -1,21 +1,52 @@
-from app.sportEvaluator import weight
+from app.sportEvaluator import getRatingModel
 from app.api.weather import liveWeather
 from app.models import weatherSummaryModel, weatherEvalModel
-from flask import Blueprint, jsonify
+from flask import Blueprint, jsonify, abort
 from flask_cors import CORS
+from app import app
 
 rating_bp = Blueprint('rating', __name__)
 CORS(rating_bp)
 
+supportedSports = app.config['SPORTS']
+
+@rating_bp.route('/ratings/<sport>')
+def get_rating(sport):
+
+    if sport.lower() not in supportedSports:
+        return abort(400, "We don't do that here")
```

Comment:
At the time figured bad request was more appropriate since the url is valid, we just don't have that piece of info. But after going through some documentation I agree that 404 is probably better.  Updates coming